main.py

```diff
@@ -1,139 +1,147 @@
-# Command Line Processing
-from genericpath import exists
-from http import client
-from importlib.resources import path
-from sys import argv
+
 import os
 import UMutils
 import UMViz
 import pandas as pd
 
-script, datahunt, iaa, schema, results, goldstandard = argv
-
-relative_dir = os.path.abspath(os.getcwd()) + "/"
-datahunt_dir = os.path.join(relative_dir, datahunt)
-iaa_dir = os.path.join(relative_dir, iaa)
-schema_dir = os.path.join(relative_dir, schema)
-results_dir = os.path.join(relative_dir, results)
-goldstandard_dir = os.path.join(relative_dir, goldstandard)
-
-import errors
-import sys
-
-try:
-    if not (
-        os.path.exists(datahunt_dir)
-        and os.path.exists(schema_dir)
-        and os.path.exists(results_dir)
-    ):
-        raise errors.InvalidDataDirectoryError
-except errors.InvalidDataDirectoryError:
-    print("One of the required data directories does not exist.")
-    sys.exit()
-
-import pymysql
-import time
-
 # Configuration Settings
-from dotenv import load_dotenv
-
-load_dotenv()
-
-endpoint = os.environ.get("ENDPOINT")
-username = os.environ.get("USERNAME")
-password = os.environ.get("PASSWORD")
-database_name = os.environ.get("DB_NAME")
-
-# LambdaHandler API to interact with MySQL database
-import LambdaHandlers
-
-api = LambdaHandlers.LambdaHandler(
```

Comment:
THis stuff isn't really deleted, just moved to a function so I could run it easier from PYCharm